spec/rubocop/cop/layout/end_of_line_spec.rb

```diff
@@ -1,56 +1,63 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Layout::EndOfLine, :config do
+  let(:message)          { 'Carriage return character missing.' }
```

Comment:
Could you please copy paste these messages instead? Most specs are like this. Note you can abbreviate with `[...]` when the message is long.